Source/Libraries/openXDA.APIAuthentication/APIAuthenticationMiddleware.cs

```diff
@@ -67,14 +69,28 @@ public AuthorizationHeader(IHeaderDictionary headers)
                 byte[] credentialData = Convert.FromBase64String(Credentials);
                 string decode = Encoding.UTF8.GetString(credentialData);
                 int index = decode.IndexOf(':');
+                int impersonationIndex = decode.LastIndexOf(':');
```

Comment:
Technically the `:` is already used to delineate `APIKey` and `APIToken` so a second `:` would most likely not cause additional Issues. I would prefer to keep it in the `Authorization` Header since that's a standard Header.
We could use a separate Type (instead of `XDA-API` use `XDA-API-IMP`?

Reply:
I'm okay with `XDA-API-IMP`.